meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
+* SC: People are welcome to join the CG or if any existing members want to collaborate with them that's fine. 
+* eP: I'd expect that'll be Justin and maybe I will help. 
+* SC: Ok. I won't merge now. 
+* TBL: There's a person on the W3C team. He couldn't go to the ??? conference. He sat and read all the abstracts and said "now I don't have to go to the ??? conference" because he relied on the fact that the abstract can save you ??? if you have more time you can read through. Maybe you can get ChatGPT's eyes on it.
+* SC: I'd be happy to be replaced by a script. 
 
 ### Too Slow: Please provide Updates-Via shortcut to secure web socket address
 URL: https://github.com/solid/notifications/issues/110
 
 * SC: Call for response to questions 1-5 in comment: https://github.com/solid/notifications/issues/110#issuecomment-1406336400
 * SC: Let me know if clarification needed.
 * SC: Resolving this issue may help with the TR/2022/notifications-protocol-20221231 release, and moving that forward can help with the TR/2022/protocol-20221231 release:
-
-#### Add TR/2022/notifications-protocol-20221231
-URL: https://github.com/solid/specification/pull/491
-
-
-#### Add TR/2022/protocol-20221231
-URL: https://github.com/solid/specification/pull/492
-
-
+  * https://github.com/solid/specification/pull/491
+  * https://github.com/solid/specification/pull/492
+* TBL: You said to look at two questions. Is one of those "is ??? always bad"? and the other one was about caching issues with metadata? 
+* SC: I can summarize. As I understood how people's assumptions are, I see no security differences with either two approaches. I am asking for people to point to me why is one less secure. It seems one of the approaches has caching issues that need to be taken into account. The HEAD response seems to have more of a caching issue than POST on a subscription service. No security differences, yes a difference in caching. 
+* TBL: I disagree that there is no difference when you have sensitive information in the URL. Anytime the client contacts the server sensitive information will be coming back ??? 
+* SC: Both are authorized requests, if either are not they will get a 403. 
+* TBL: These happen thousands of times in option A and much less in option B. 
+* eP: Is this a roadblock for the PR or something we want to resolve for next version? 
+* SC: It is not but hoping it can help move. The PR is there and there are zero reviews. Only issue I can find to move the PR is either accept PR for next release or refuse this publication. People have different positions. If they want to battle out how it should work they should hash it out and come back to me. I don't see any security differences. The questions are there. Just need to be responded. 
```

Comment:
```suggestion
* eP: Is this a roadblock for the PR, or something we want to resolve for next version? 
* SC: It is not a roadblock, but hoping it can help move. The PR is there, and there are zero reviews. Only way I can find to move the PR is to either accept PR for next release, or refuse this publication. People have different positions. If they want to battle out how it should work, they should hash it out, and come back to me. I don't see any security differences. The questions are there. Just need to be responded to.
```